production/helm/loki/src/dashboards/loki-chunks.json

```diff
@@ -0,0 +1,1336 @@
+{
```

Comment:
I'm wondering. Shouldn't we use the dashboards from the mixin?

Reply:
Are you suggesting we instead compile them as part of the build process? I think that would be a great addition, but that we should tackle that in a later PR.

For now these are the pre-compiled version, pulled directly from: https://github.com/grafana/loki/tree/main/production/loki-mixin-compiled-ssd